test/test_bsonjs.py

```diff
@@ -291,6 +291,25 @@ def test_load_throws_no_read_attribute(self):
         not_file = {}
         self.assertRaises(AttributeError, bsonjs.load, not_file)
 
+    def test_extended(self):
```

Comment:
We can rename this to `test_mode` now. Can you also add a test case for dump (not dumps) and also add a test for passing the mode as a non-keyword (ie positional) argument?:
```
bsonjs.dumps(bson_bytes, bsonjs.CANONICAL)
```

Reply:
Did you add the test for dump()?